wordpress-mobile/WordPress-iOS
# Update strings for base en_US (canceling/canceled)

Hello,

Updated cancelled/cancelling to en_US canceled/canceling so Glossaries pick up these strings and to have base locale american.

I didn't touch any variable names and focused on string changes. There was a few comments I did change but those are ok to revert.

I wasn't sure if I’m supposed to leave Base.lproj/Localizable.strings one untouched?
https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/WordPressShareExtension/Base.lproj/Localizable.strings
*If that's a generated file then I can remove the change made there.

To test:
Generate strings and make sure they don't contain cancelled or cancelling and are instead the american version of canceled and canceling.

Update release notes:
